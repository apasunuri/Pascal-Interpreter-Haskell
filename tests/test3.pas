program test3;

var
    temp_f, temp_c : real;

begin
    temp_f := 25;
    temp_c := (5 * (temp_f - 32)) / 9;
    writeln('The Weather in Fahrenheit');
    writeln(temp_f);
    writeln('Temperature in Celsius');
    writeln(temp_c);
    if(temp_c <= 0) then
        writeln('It is below freezing temperature.')
    else
        writeln('It is above freezing temperature.');
end.