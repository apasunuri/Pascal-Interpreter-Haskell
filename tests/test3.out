Enter the Weather in Fahrenheit
25.0
Temperatures in Celsius
-3.8888888
It is below freezing temperature.